application/engine/services/autopilot_daemon.py

```
@@ -338,6 +338,16 @@ async def _handle_act_planning(self, novel: Novel):
 
         target_act = next((n for n in act_nodes if n.number == target_act_number), None)
 
+        # 结构推荐参数：在动态幕生成与章节规划 fallback 两个分支都会用到，
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)